packages/airnode-deployer/src/cli/commands.test.ts

```diff
@@ -1,25 +1,34 @@
-import { join } from 'path';
+import fs from 'fs';
```

Comment:
Nit: Can we enforce it with eslint? I prefer the member import syntax, but I am OK with this if it's for the consistency sake.